modules/sc-mesh-secure-deployment/src/1_5/common/mesh_utils.py

```diff
@@ -11,6 +12,20 @@
 except ImportError:
     import Queue as queue
 
+def get_mesh_int():
```

Comment:
get_mesh_intf() or get_mesh_interface() could be better function name as *_int() - is a bit misleading.